### PR TITLE
Add bazfire-jan hometown file

### LINKED ISSUE #13: Practice Github Flow - bazfire
## Create Check List
- [] Create an issue
- [] Create an issue
- [] Create an issue
- [] Create an issue
- [] Create an issue
- [] Create an issue

### PR DESCRIPTION
**Pull request by bazfire**
This is to add an overview of things to do in jan. 

closes #13

Addresses #13 
Fixes #13

### CODE REVIEW

**Comment by bazfire** [comment]
@crichID can you please take  look and let me know what you think?

**Comment by githubteacher** [comment]
@bazfire, could you please add fun things to do in your hometown?

**Comment by crichID** [comment]
> can you please take look and let me know what you think?

@bazfire looks great so far! Looking forward to seeing more!

**Comment by bazfire** [comment]
@githubteacher check out my changes

File: bazfire-jan.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Fun Things to Do 
+
+- [] Paddling on the lake
```
Comment (mflorys) [inline bazfire-jan.md]: is it safe?

File: bazfire-jan.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## Fun Things to Do 
+
+- [] Paddling on the lake 
+- [] Mount Bonell
```
Comment (mflorys) [inline bazfire-jan.md]: replace "ll" with "r" please

Comment (bazfire) [inline bazfire-jan.md]: Nah!

**Pull request merged**